app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/LectureViewDrawer.kt

```diff
@@ -0,0 +1,237 @@
+package nerd.tuxmobil.fahrplan.congress.schedule
+
+import android.content.Context
+import android.content.res.Configuration
+import android.content.res.Resources
+import android.graphics.Typeface
+import android.preference.PreferenceManager
+import android.support.annotation.ColorInt
+import android.support.annotation.ColorRes
+import android.support.v4.content.ContextCompat
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.ImageView
+import android.widget.LinearLayout
+import android.widget.TextView
+import info.metadude.android.eventfahrplan.commons.temporal.Moment
+import nerd.tuxmobil.fahrplan.congress.MyApp
+import nerd.tuxmobil.fahrplan.congress.R
+import nerd.tuxmobil.fahrplan.congress.contract.BundleKeys
+import nerd.tuxmobil.fahrplan.congress.extensions.getLayoutInflater
+import nerd.tuxmobil.fahrplan.congress.models.Lecture
+import org.threeten.bp.Duration
+
+internal class LectureViewDrawer(context: Context, private val fahrplanFragment: FahrplanFragment) {
```

Comment:
Same as with https://github.com/EventFahrplan/EventFahrplan/pull/256#discussion_r404126003

Reply:
I respect that you try "to only ever change as few logic as possible in this PR". Nevertheless, I want to encourage to apply best practice whenever a new file or class is created. This might be through extracting code or refactoring existing code (to Kotlin). As mentioned, the auto-conversion of Android Studio only does a "okay" job - manual changes are needed.

Here I find that passing a reference to a `Fragment` class is too much. I would not expose the large interface of `FahrplanFragment` to `LectureViewDrawer` because at a minimum only to callbacks are needed.

Reply:
Ok, I solved it using existing interface types from View class.

Reply:
Nice. That is much better than what I suggested. Thank you for finding the interfaces.